cli/node/mod.rs

```diff
@@ -412,24 +416,21 @@ pub fn node_resolve_npm_reference(
   let package_folder = npm_resolver
     .resolve_package_from_deno_module(&reference.req)?
     .folder_path;
-  let maybe_url = package_config_resolve(
+  let resolved_path = package_config_resolve(
     &reference
       .sub_path
       .as_ref()
       .map(|s| format!("./{}", s))
       .unwrap_or_else(|| ".".to_string()),
     &package_folder,
     npm_resolver,
+    NodeModuleKind::Esm,
```

Comment:
Why is it always ESM?

Reply:
It’s the referrer kind, which is an npm package reference, which is deno code, so esm.